addon/components/au-date-picker.js

```diff
@@ -1,93 +1,142 @@
 import Component from '@glimmer/component';
 import { action } from "@ember/object";
-import { tracked } from "@glimmer/tracking";
+import { assert, runInDebug } from '@ember/debug';
+import { guidFor } from '@ember/object/internals';
+
+const DEFAULT_ADAPTER = {
+  parse: function(value = '', createDate) {
+    const matches = value.match(/^(\d{1,2})-(\d{1,2})-(\d{4})$/);
+    if (matches) {
+      return createDate(matches[3], matches[2], matches[1]);
+    }
+  },
+  format: function(date) {
+    return `${date.getDate()}-${date.getMonth() + 1}-${date.getFullYear()}`;
+  },
+};
+
+const DEFAULT_LOCALIZATION = {
+  buttonLabel: 'Choose date',
+  placeholder: 'DD-MM-YYYY',
+  selectedDateMessage: 'Selected date is',
+  prevMonthLabel: 'Previous month',
+  nextMonthLabel: 'Next month',
+  monthSelectLabel: 'Month',
+  yearSelectLabel: 'Year',
+  closeLabel: 'Close window',
+  keyboardInstruction: 'You can use arrow keys to navigate dates',
+  calendarHeading: 'Choose a date',
+  dayNames: ['Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday'],
+  monthNames: ['January', 'February', 'March', 'April', 'May', 'June', 'July', 'August', 'September', 'October', 'November', 'December'],
+  monthNamesShort: ['Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun', 'Jul', 'Aug', 'Sep', 'Oct', 'Nov', 'Dec'],
+};
 
 export default class AuDatePickerComponent extends Component {
+  @isoDate value;
```

Comment:
`@isoDate` might not be the clearest name but couldn't think of something else.

Reply:
perhaps `dateRepresentation` or `isoDateRepresentation`?

Reply:
Renamed to `isoDateRepresentation`.